pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+Please fill in the release notes. See [README](https://github.com/prestodb/presto/blob/master/README.md#release-notes) for details.
+
+    == RELEASE NOTES ==
```

Comment:
Is this going to be added automatically when I add commit message? Can we have checkstyle to enforce this?

Reply:
When this is checked in, anyone who create a PR will have this pre-populated in the PR description. We're looking for release at PR level, not at commit level.

There should be a way to setup tests that will fail if PR does not include such a release note section, but I'm not prioritizing in looking into that right now. I have a script in presto-facebook to collect all the release notes which will be used by release oncalls and it will handle all the cases. I'll send that our shortly.

Reply:
Why does that need to be in presto-facebook?

Reply:
Currently our release-related scripts are all in presto-facebook. I make sense to make it open-source it, but maybe in a separate Python repo.

Reply:
Thanks for the clarification.